sql/catalyst/src/main/scala/org/apache/spark/sql/connector/expressions/expressions.scala

```diff
@@ -389,3 +390,8 @@ private[sql] object SortValue {
       None
   }
 }
+
+private[sql] abstract class AbstractExpression extends Expression with Serializable {
```

Comment:
I mean java package private... basically just define the class without `public`. We can simply put this class in `Expression.java`

Reply:
For the name, how about `ExpressionWithToString`?

Reply:
I got it.